microsoft-edge/webview2/concepts/overview-features-apis.md

```diff
@@ -1120,6 +1122,38 @@ See also:
 * `ICoreWebView2Settings8` interface:
    * [ICoreWebView2Settings8::get_IsReputationCheckingRequired method](/microsoft-edge/webview2/reference/win32/icorewebview2settings8#get_isreputationcheckingrequired)
    * [ICoreWebView2Settings8::put_IsReputationCheckingRequired method](/microsoft-edge/webview2/reference/win32/icorewebview2settings8#put_isreputationcheckingrequired)
+
+---
+
+
+<!-- ---------- -->
+###### Custom crash reporting
+
+<!-- todo: first-time intro for Dev students: -->
+Custom crash reporting.
```

Comment:
@dianaqu can you please add a 1-2 sentence blurb about this API?

Reply:
clickable links:
.NET:
* [CoreWebView2EnvironmentOptions.IsCustomCrashReportingEnabled Property](http://learn.microsoft.com/dotnet/api/microsoft.web.webview2.core.corewebview2environmentoptions.iscustomcrashreportingenabled)
* [CoreWebView2Environment.FailureReportFolderPath Property](http://learn.microsoft.com/dotnet/api/microsoft.web.webview2.core.corewebview2environment.failurereportfolderpath)
WinRT:
* [CoreWebView2EnvironmentOptions.IsCustomCrashReportingEnabled Property](http://learn.microsoft.com/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environmentoptions#iscustomcrashreportingenabled)
* [CoreWebView2Environment.FailureReportFolderPath Property](http://learn.microsoft.com/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environment#failurereportfolderpath)
Win32:
* [ICoreWebView2EnvironmentOptions3::get_IsCustomCrashReportingEnabled](http://learn.microsoft.com/microsoft-edge/webview2/reference/win32/icorewebview2environmentoptions3#get_iscustomcrashreportingenabled)
* [ICoreWebView2Environment11::get_FailureReportFolderPath](http://learn.microsoft.com/microsoft-edge/webview2/reference/win32/icorewebview2environment11#get_failurereportfolderpath)
Spec: 
https://github.com/MicrosoftEdge/WebView2Feedback/blob/main/specs/DisableCrashReporting.md

Reply:
another PR temporarily had heading "Data collection" rather than "Custom crash reporting"

Reply:
If any WebView2 process crashes, one or multiple minidump files will be created and sent to Microsoft for diagnosis. This API allow the end developer to customize crash reporting to help when running diagnostics and doing analysis. They can set the `CoreWebView2EnvironmentOptions.IsCustomCrashReportingEnabled` property to true to prevent crash dumps from being sent to Microsoft and use the `CoreWebView2Environment.CrashDumpFolderPath` property to locate crash dumps and do customization with them instead.

Reply:
incorp'd